smart-frontend/app/src/components/pages/RulesPage/useRequestRules.ts

```diff
@@ -41,6 +47,8 @@ export const useRequestRules = () => {
   const debounceGetData = useCallback(
     useDebounce(() => {
       dispatch(getRules());
+      dispatch(getAllRulesCount());
```

Comment:
we shoul update meta info for `refreshRules` too

Reply:
Added to refresh